aptos-move/framework/aptos-stdlib/Move.toml

```diff
@@ -5,6 +5,7 @@ version = "1.0.0"
 [addresses]
 std = "0x1"
 aptos_std = "0x1"
+Extensions = "0x1" # TODO: Needed for Prover to instantiate `{{Ext}}` in prelude.
```

Comment:
Why is this a TODO? It sounds like the prover just needs it.

Reply:
I will see if there is a better way to handle this in Prover.